lib/Game.js

```diff
@@ -1,9 +1,10 @@
 class Game {
-  constructor(player1, player2, player3, round = 1) {
+  constructor(players, round = 1) {
     this.players = [];
```

Comment:
It looks like you're passing through players as a parameter but you're defaulting `this.players` to an empty array. What are you doing with the `players` that you're passing through? Right now it seems to be unused.